cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -22,28 +22,34 @@ function _port() {
 function prepare_config() {
     BASE_PATH=${KUBEVIRTCI_CONFIG_PATH:-$PWD}
     cat >$BASE_PATH/$KUBEVIRT_PROVIDER/config-provider-$KUBEVIRT_PROVIDER.sh <<EOF
-master_ip=$(_main_ip)
-kubeconfig=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
-kubectl=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubectl
-gocli=${BASE_PATH}/../cluster-up/cli.sh
-docker_prefix=\${DOCKER_PREFIX:-localhost:$(_port registry)/kubevirt}
-manifest_docker_prefix=\${DOCKER_PREFIX:-registry:5000/kubevirt}
-EOF
+    master_ip=$(_main_ip)
+    kubeconfig=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
+    kubectl=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubectl
+    gocli=${BASE_PATH}/../cluster-up/cli.sh
+    docker_prefix=\${DOCKER_PREFIX:-localhost:$(_port registry)/kubevirt}
+    manifest_docker_prefix=\${DOCKER_PREFIX:-registry:5000/kubevirt}
+    EOF
 }
 
 function _registry_volume() {
     echo ${job_prefix}_registry
 }
 
 function _add_common_params() {
-    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE} --cpu 5 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
```

Comment:
this change will become unneeded once 
https://github.com/kubevirt/kubevirtci/pull/223
will be merged, because 223 fixes it globally

Reply:
removed it from here, tested locally k8s